enclave/core/globals.c

```diff
@@ -18,13 +131,17 @@ extern volatile const oe_sgx_enclave_properties_t oe_enclave_properties_sgx;
 
 const void* __oe_get_enclave_base()
 {
-    /*
-     * Note: The reference to &oe_enclave_properties_sgx will be compiled
-     * IP-relative by the C-compiler on x86_64, and hence does not have a
-     * relocation entry. Thus it works both pre- and post-relocation.
-     */
+/*
```

Comment:
Is the comment for Windows or Linux?

Reply:
Moved this comment to windows branch of the #if